src/librbd/cache/pwl/ssd/WriteLog.cc

```diff
@@ -640,6 +640,7 @@ bool WriteLog<I>::retire_entries(const unsigned long int frees_per_tx) {
             this->m_blocks_to_log_entries.remove_log_entry(write_entry);
           }
         }
+	first_valid_entry = control_bloc_pos + data_length + MIN_WRITE_ALLOC_SSD_SIZE;
```

Comment:
Did you compile this?

Reply:
Sorry for forget add WITH_RBD_SSD_CACHE. Update.